pyproject.toml

```diff
@@ -126,6 +118,19 @@ ignore = [
   "F811",
 ]
 
+[tool.ruff.lint.per-file-ignores]
+# Ignore docstring rules everywhere except for the stable files (particleset.py).
+"!src/parcels/_core/{particleset}.py" = [
+  # Missing docstring in public module
+  "D100",
+  # Missing docstring in public class
+  "D101",
+  # Missing docstring in public method
+  "D102",
+  # Missing docstring in public function
+  "D103",
+]
```

Comment:
Looks good. Up to you whether you'd like to manage this in the same file this way, or via a separate file - either works